.travis.yml

```diff
@@ -21,6 +21,11 @@ env:
   matrix:
     - ROS_REPO=ros
     - ROS_REPO=testing
+    - ROS_REPO="testing" \
+      CATKIN_LINT=false NOT_TEST_BUILD=true \
+      AFTER_SCRIPT="./.coverage.sh" \
+      CMAKE_ARGS="-DENABLE_COVERAGE_TESTING=ON -DCMAKE_BUILD_TYPE=Debug" \
+      DOCKER_RUN_OPTS='-e COVERAGE_PKGS="prbt_hardware_support"'
```

Comment:
is there a cleaner way?

Reply:
I am open for suggestions.

Reply:
I just thought it may be nicer to somehow wrap all of them into a file but that may increase the complexity unnecessarily and may not even be possible. So I guess we leave it as it is.